src/lib/environment.ts

```diff
@@ -20,6 +22,10 @@ export const IS_ANDROID = !!(userAgentLowerCase.includes('android')
 
 export const IS_MOBILE_DEVICE = userAgentLowerCase.includes('mobi');
 
+export const IS_CHROME = userAgentLowerCase.includes('chrome');
```

Comment:
what about Chrome based browsers?

Reply:
I renamed the constant to be more precise. Is this OK in your opinion?